client/scss/components/_panel.scss

```diff
@@ -32,6 +32,10 @@ $header-gap: theme('spacing.1');
   cursor: pointer;
   padding-inline-end: theme('spacing.2');
 
+  label {
```

Comment:
Not sure whether to add this nested style or just change the selector to `.w-panel__heading, .w-panel__heading label`

Reply:
This seems ok to me.